063-moving-data-between-two-jbod-disks-using-cruise-control.md

```diff
@@ -0,0 +1,71 @@
+# Moving data between two JBOD disks using Cruise Control
+
+This proposal is about integrating the `remove_disks` endpoint from Cruise Control into Strimzi cluster operator. 
+This endpoint will allow us to move the data between two JBOD disks. 
+
+## Current situation
+
+Currently, we get a multiple requests to add the ability for moving all Kafka logs between two disks on the JBOD storage array. This feature can be useful in following scenarios:
+- The current disk is too big and the user wants to use smaller one
+- When we want to use different Storage Class with different parameters or different storage types.
+For now, we can do this using the Kafka CLI tools but it is not very user-friendly.
```

Comment:
which Kafka CLI tools?

Reply:
I guess we want to mention `kafka-reassign-partitions.sh` tool